fastlane/Fastfile

```diff
@@ -63,27 +66,27 @@ before_all do |lane|
 end
 
 platform :ios do
-########################################################################
-# Environment
-########################################################################
-Dotenv.load(USER_ENV_FILE_PATH)
-Dotenv.load(PROJECT_ENV_FILE_PATH)
-ENV[GHHELPER_REPO="wordpress-mobile/wordpress-iOS"]
-ENV["PROJECT_NAME"]="WordPress"
-ENV["PUBLIC_CONFIG_FILE"]=File.join(PROJECT_ROOT_FOLDER, "config", "Version.Public.xcconfig")
-ENV["INTERNAL_CONFIG_FILE"]=File.join(PROJECT_ROOT_FOLDER, "config", "Version.internal.xcconfig")
-ENV["DOWNLOAD_METADATA"]="./fastlane/download_metadata.swift"
-ENV["PROJECT_ROOT_FOLDER"]=PROJECT_ROOT_FOLDER + "/"
-ENV["APP_STORE_STRINGS_FILE_NAME"]="AppStoreStrings.po"
-
-########################################################################
-# Screenshots
-########################################################################
-import "./ScreenshotFastfile"
-
-########################################################################
-# Release Lanes
-########################################################################
+  ########################################################################
+  # Environment
+  ########################################################################
+  Dotenv.load(USER_ENV_FILE_PATH)
+  Dotenv.load(PROJECT_ENV_FILE_PATH)
+  ENV[GHHELPER_REPO = 'wordpress-mobile/wordpress-iOS']
+  ENV['PROJECT_NAME'] = 'WordPress'
+  ENV['PUBLIC_CONFIG_FILE'] = File.join(PROJECT_ROOT_FOLDER, 'config', 'Version.Public.xcconfig')
+  ENV['INTERNAL_CONFIG_FILE'] = File.join(PROJECT_ROOT_FOLDER, 'config', 'Version.internal.xcconfig')
+  ENV['DOWNLOAD_METADATA'] = './fastlane/download_metadata.swift'
+  ENV['PROJECT_ROOT_FOLDER'] = PROJECT_ROOT_FOLDER
```

Comment:
Good point. When I made this change in isolation in 52ee45a, I [grepped for `PROJECT_ROOT_FOLDER`](https://github.com/wordpress-mobile/release-toolkit/search?q=PROJECT_ROOT_FOLDER&type=).

```
➜ g grep PROJECT_ROOT
lib/fastlane/plugin/wpmreleasetoolkit/actions/android/android_download_translations_action.rb:        res_dir = File.join(ENV['PROJECT_ROOT_FOLDER'] || '.', params[:res_dir])
lib/fastlane/plugin/wpmreleasetoolkit/actions/android/android_update_release_notes.rb:        path = File.join(ENV['PROJECT_ROOT_FOLDER'] || '.', 'RELEASE-NOTES.txt')
lib/fastlane/plugin/wpmreleasetoolkit/actions/ios/ios_update_release_notes.rb:        path = File.join(ENV['PROJECT_ROOT_FOLDER'] || '.', 'RELEASE-NOTES.txt')
lib/fastlane/plugin/wpmreleasetoolkit/helper/android/android_git_helper.rb:        # @env PROJECT_ROOT_FOLDER The path to the git root of the project
lib/fastlane/plugin/wpmreleasetoolkit/helper/android/android_git_helper.rb:            files: File.join(ENV['PROJECT_ROOT_FOLDER'], ENV['PROJECT_NAME'], 'build.gradle'),
lib/fastlane/plugin/wpmreleasetoolkit/helper/android/android_git_helper.rb:        # @env PROJECT_ROOT_FOLDER The path to the git root of the project
lib/fastlane/plugin/wpmreleasetoolkit/helper/android/android_git_helper.rb:          res_dir = File.join(ENV['PROJECT_ROOT_FOLDER'], ENV['PROJECT_NAME'], 'src', 'main', 'res')
lib/fastlane/plugin/wpmreleasetoolkit/helper/android/android_version_helper.rb:          gradle_file_path = File.join(ENV['PROJECT_ROOT_FOLDER'] || '.', 'build.gradle')
lib/fastlane/plugin/wpmreleasetoolkit/helper/android/android_version_helper.rb:        # @env PROJECT_ROOT_FOLDER The path to the root of the project (the folder containing the `.git
lib/fastlane/plugin/wpmreleasetoolkit/helper/android/android_version_helper.rb:          UI.user_error!("You need to set the \`PROJECT_ROOT_FOLDER\` environment variable to the path
lib/fastlane/plugin/wpmreleasetoolkit/helper/android/android_version_helper.rb:          File.join(ENV['PROJECT_ROOT_FOLDER'], ENV['PROJECT_NAME'], 'build.gradle')
lib/fastlane/plugin/wpmreleasetoolkit/helper/ios/ios_git_helper.rb:        # @env PROJECT_ROOT_FOLDER The path to the git root of the project
lib/fastlane/plugin/wpmreleasetoolkit/helper/ios/ios_git_helper.rb:          files_list = [File.join(ENV['PROJECT_ROOT_FOLDER'], 'config', '.')]
lib/fastlane/plugin/wpmreleasetoolkit/helper/ios/ios_git_helper.rb:            files_list.append File.join(ENV['PROJECT_ROOT_FOLDER'], ENV['PROJECT_NAME'], 'Resources', ENV['APP_STOR
lib/fastlane/plugin/wpmreleasetoolkit/helper/ios/ios_git_helper.rb:        # @env PROJECT_ROOT_FOLDER The path to the git root of the project
lib/fastlane/plugin/wpmreleasetoolkit/helper/ios/ios_git_helper.rb:          Action.sh("cd #{get_from_env!(key: 'PROJECT_ROOT_FOLDER')} && ./Scripts/localize.py")
lib/fastlane/plugin/wpmreleasetoolkit/helper/ios/ios_git_helper.rb:        # @env PROJECT_ROOT_FOLDER The path to the git root of the project
lib/fastlane/plugin/wpmreleasetoolkit/helper/ios/ios_git_helper.rb:          Action.sh("cd #{get_from_env!(key: 'PROJECT_ROOT_FOLDER')} && ./Scripts/update-translations.rb")
lib/fastlane/plugin/wpmreleasetoolkit/helper/ios/ios_git_helper.rb:          project_root = get_from_env!(key: 'PROJECT_ROOT_FOLDER')
lines 1-19/19 (END)
```

AFAICS, all the instances are used within `File.join` which handles the `/`. That includes the [assignment to `project_root`](https://github.com/wordpress-mobile/release-toolkit/blob/develop/lib/fastlane/plugin/wpmreleasetoolkit/helper/ios/ios_git_helper.rb#L65-L70) in the last line of the `git grep` output.

Can you see anything that I missed? Otherwise, how do you feel about risking it and see if it breaks anything? (I could do it once I roll over as release manager)

I'm saying it mainly because removing the `/` is the right thing to do™

Reply:
> how do you feel about risking it and see if it breaks anything

I'd be ok with it, but then we're committing to a major version bump of this library – do we want that?

IMHO, while it is more correct to adjust this, I'd prefer to make a larger change to not require these global variables anymore (even if it's "not right" for a while).

If you're ok with bumping the major version we can do the change though 😃

Reply:
IMHO it's a cosmetic issue, so while it's not correct, I'd _personally_ prefer to address the structural problem (global variables) in the release toolkit? I'd be concerned about the potential for things silently breaking during a transition.

That said, you're the one who'd have to catch them, not me, so maybe just give it a think and feel free to go either way? 🙂

Reply:
I was tempted to say "Yeah no worries. I checked the code and I'm sure that won't be any problems." Then I had visions of broken code freezes and tears of frustration.

Reverted! fb23430 (🙌 past me for small atomic commits)

Thank you for being wise.